DEVELOP.md

```diff
@@ -64,24 +64,22 @@ make build
 make release
 ```
 
-## Layer0 Runner
-The Layer0 Runner is a tool that runs asynchronous jobs.
-The [Layer0 API](#layer0-api) creates job for the Layer0 Runner.
+## Layer0 Terraform plugin
```

Comment:
I know we mentioned terraform version dependency as a client-facing requirement, but I still think we should call out what version we're using for our own development as well.

Reply:
added